index.html

```diff
@@ -31,8 +31,8 @@
         <div class="app">
             <header>
                 <span class="header-contents">
-                    <a id="logo" href="#" title="Swift Evolution" role="img" aria-label="Swift Evolution"></a>
-                    <h1 id="title"><a href="#">Swift Evolution</a></h1>
+                    <a id="logo" href="." title="Swift Evolution" role="img" aria-label="Swift Evolution"></a>
```

Comment:
Although from a coding perspective it’s expedient to do this, does it really make sense to reload the entire page just to toggle off the filter?

Seems like we should add a function for this, roughly: 

```javascript
if(!document.querySelector('.filter-toggle').classList.contains('hidden')) { toggleFiltering() }
```

Reply:
Currently, When clicking on logo/title, query string in URL has been removing but there are some filter still

Reply:
Hmm, fair point. Let’s just go with your original approach, then.